src/fees/compute_fees.py

```diff
@@ -436,48 +478,35 @@ def parse_fee_policies(
         return fee_policies
 
 
-# computing fees
-def compute_fee_imbalances(
-    settlement_data: SettlementData,
-) -> tuple[dict[str, tuple[str, int]], dict[str, tuple[str, int]]]:
+# function that computes all fees of all orders in a batch
+# Note that currently it is NOT working for CoW AMMs as they are not indexed.
```

Comment:
Does that mean that the code cannot be used for payments? What is required to make it work? Will if just be automatically correct after jit orders appear on the orders endpoint?

Reply:
Yes, for jit CoW AMM orders we currently cannot compute network fees. This will hopefully be resolved in the next 1-2 weeks as executed CoW AMM orders will appear in the endpoint.

Reply:
Actually backend reported this is happening in prod next week so we should be fine soon.